osmpy/core.py

```diff
@@ -12,8 +12,9 @@
 import pandas as pd
 import warnings
 import re
-import osmpy.queries as qs
+from .queries import *
```

Comment:
I'd rather use explicit imports.

Maybe `from .queries import QueryType` solves it

Reply:
I revised these parts because when I cloned the repository, some imports didn't work when installed locally.
Here, I'm actually importing not just `QueryType`, but all the classes inherited from it.
As an alternative, you can keep the original imports and `_get_queries_names()`, just test it and make sure everything works fine.